rbx_dom_lua/src/database.json

```diff
@@ -47744,11 +49925,17 @@
         "Attributes": {
           "Attributes": {}
         },
+        "Capabilities": {
+          "SecurityCapabilities": 0
+        },
+        "DefinesCapabilities": {
+          "Bool": false
+        },
         "Index": {
           "Int32": -1
         },
         "PropertiesSerialize": {
-          "BinaryString": ""
+          "BinaryString": "AAAAAA=="
```

Comment:
This is something to keep an eye on, since I imagine we'll want to care about `StyleRule`s